app/src/main/res/layout/fragment_scheduler.xml

```diff
@@ -8,189 +8,272 @@
+                        fab:fab_size="mini"/>
+
+                    <com.github.clans.fab.FloatingActionButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:src="@drawable/ic_smile_24dp"
+                        app:rippleColor="@color/color_white"
+                        bind:ignore="VectorDrawableCompat"
+                        fab:fab_colorNormal="@color/color_white"
+                        fab:fab_label="@string/tilte_finished"
+                        fab:fab_size="mini"/>
 
-            <com.github.clans.fab.FloatingActionButton
+                    <com.github.clans.fab.FloatingActionButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:src="@drawable/ic_broken_heart_24dp"
+                        bind:ignore="VectorDrawableCompat"
+                        fab:fab_colorNormal="@color/color_white"
+                        fab:fab_label="@string/title_cancel"
+                        fab:fab_size="mini"/>
+
+                    <com.github.clans.fab.FloatingActionButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:src="@drawable/ic_waiting_room_24_dp"
+                        bind:ignore="VectorDrawableCompat"
+                        fab:fab_colorNormal="@color/color_white"
+                        fab:fab_label="@string/title_waiting"
+                        fab:fab_size="mini"/>
+                </com.github.clans.fab.FloatingActionMenu>
+            </android.support.design.widget.CoordinatorLayout>
+
+            <android.support.design.widget.NavigationView
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:src="@drawable/ic_waiting_room_24_dp"
-                bind:ignore="VectorDrawableCompat"
-                fab:fab_colorNormal="@color/color_white"
-                fab:fab_label="@string/title_waiting"
-                fab:fab_size="mini"/>
-        </com.github.clans.fab.FloatingActionMenu>
-    </android.support.design.widget.CoordinatorLayout>
+                android:layout_height="match_parent"
+                android:layout_gravity="start"
+                android:background="@color/color_white">
+
+                <android.support.v4.widget.NestedScrollView
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    >
+
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:orientation="vertical"
+                        android:paddingLeft="@dimen/dp_10"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingLeft` you should probably also define `paddingRight` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.